src/gt4py/next/ffront/fbuiltins.py

```diff
@@ -243,19 +263,27 @@ def impl(value: Field | gt4py_defs.ScalarT, /) -> Field | gt4py_defs.ScalarT:
 ):
     _make_unary_math_builtin(f)
 
-BINARY_MATH_NUMBER_BUILTIN_NAMES = ["minimum", "maximum", "fmod", "power"]
+BINARY_MATH_NUMBER_BUILTIN_TO_PYTHON_SCALAR_FUNCTION = {
+    "minimum": min,
+    "maximum": max,
+    "fmod": math.fmod,
+    "power": pow,
+}
+BINARY_MATH_NUMBER_BUILTIN_NAMES = list(BINARY_MATH_NUMBER_BUILTIN_TO_PYTHON_SCALAR_FUNCTION.keys())
 
 
 def _make_binary_math_builtin(name):
     def impl(
-        lhs: Field | gt4py_defs.ScalarT,
-        rhs: Field | gt4py_defs.ScalarT,
+        lhs: common.Field | core_defs.ScalarT,
+        rhs: common.Field | core_defs.ScalarT,
         /,
-    ) -> Field | gt4py_defs.ScalarT:
-        raise NotImplementedError()
+    ) -> common.Field | core_defs.ScalarT:
+        assert core_defs.is_scalar_type(lhs)
+        assert core_defs.is_scalar_type(rhs)
+        return BINARY_MATH_NUMBER_BUILTIN_TO_PYTHON_SCALAR_FUNCTION[name](lhs, rhs)  # type: ignore[operator] # Cannot call function of unknown type
```

Comment:
I feel I'm missing something obvious. `np.minimum` is an extension of the functionality of `min`, so I don't understand your first sentence. Unless that is wrong our naming is indeed clever as we chose the numpy name for all builtins and not sometimes the python ones in case they exist.

Reply:
Ah now, I get it, you want to do the scalar functions using numpy?

Reply:
Yes :-)

Reply:
done